tests/test_regression_images.py

```diff
@@ -197,10 +195,21 @@ def test_read_image(test_image_for_reading):
 
         print("%-40s" % format_instance.__name__, R_raw_data[0].focus())
 
+        # also exercise get_corrected_data - iff it has the method
+        if hasattr(instance, "get_corrected_data"):
+
+            try:
+                R_corrected_data = instance.get_corrected_data()
```

Comment:
I don't think that `format_class.get_corrected_data()` is a thing (or at least, isn't any more). You probably need to get hold of the imageset via the format class, and then call `get_corrected_data()` on that.